apps/quick-learn-backend/project.json

```diff
@@ -5,24 +5,25 @@
   "projectType": "application",
   "tags": [],
   "targets": {
-    // "build": {
-    //   "executor": "@nx/webpack:webpack",
-    //   "outputs": [
-    //     "{options.outputPath}"
-    //   ],
-    //   "options": {
-    //     "target": "node",
-    //     "compiler": "tsc",
-    //     "outputPath": "dist/apps/quick-learn-backend",
-    //     "main": "apps/quick-learn-backend/src/main.ts",
-    //     "tsConfig": "apps/quick-learn-backend/tsconfig.app.json",
-    //     "assets": [
-    //       "apps/quick-learn-backend/src/assets"
-    //     ],
-    //     "isolatedConfig": true,
-    //     "webpackConfig": "apps/quick-learn-backend/webpack.config.js"
-    //   }
-    // },
+    "build": {
```

Comment:
The Build Option will be not. Better it should be cleaned.

Reply:
remove webpackConfig